activities/live-streaming/post-production.md

```diff
@@ -0,0 +1,24 @@
+---
+type: guide
+explains: how to do post-production a live stream
+---
+
+# Post-producing a live stream
+
+When doing post-production of a 'Let's talk about  public code' [live stream](index.md), there are some steps to remember.
+
+- Follow up of attendees/viewers: connect them to a Call to Action such as [sign up for our tailored-to-your-preferences-newsletter](https://forms.gle/gn7wR2Eaxbv5g1BF9).
+- Cut an audio version and upload to [Podbean](../tool-management/podbean.md).
+  - Cut out log pauses, coughs and other noise, but try to keep the content similiar to the recording.
+  - Compress and normalize.
+  - Cut out the long intro audio for a shorter audio intro.
+  - Change the outro audio for a complete version.
+- Post a blogpost with the link to the video and also the audio.
+- Social media campaign ending.
```

Comment:
Does this need a verb? Does it include publicly thanking the guest again?

Reply:
Hmm. Yes, it needs something more. @AlbaRoza I copied this from your notes, how can we clarify this?